views/templates.pug

```diff
@@ -89,9 +89,12 @@
           button.dropdown-item.btn.btn-sm.btn-light.view-ast(title="Show AST output")
             span.dropdown-icon.fas.fa-leaf
             | AST output
-          button.dropdown-item.btn.btn-sm.btn-light.view-ir(title="Show IR output")
+          button.dropdown-item.btn.btn-sm.btn-light.view-ir(title="Show LLVM IR output")
             span.dropdown-icon.fas.fa-align-center
-            | IR output
+            | LLVM IR output
```

Comment:
Thoughts on renaming this tab item to LLVM IR to indicate that it's specifically the LLVM IR? This list will probably get crowded with other compilers' IR representations further down the road. @ojeda mentions this in #2565

Reply:
I think being explicit is usually a positive -- and it is not too much text, so I guess it does note take too much space, no?

Reply:
Nearly all items in this dropdown menu are compiler specific. I remember mentioning this to @RubenRBS with the idea of hiding items not compatible with currently selected compiler but there was something against it. The same renaming should be done to AST (llvm), optim output (llvm) and graph output (gcc).

Reply:
So! My initial idea was to add subdropdowns or categorize the panes better, but that's more trouble than it's worth.
My initial thought was that it would be a bit tricky to add compatibility checks for all panes for every compiler, but we already do it! That's how we know when to disable to button itself. 

So, what we can do is to merge this, and find a way to only show the supported panes instead of disabling them (Might not be as easy as substituting .prop('disabled', ...) by .toggle(...), but I might also be wrong)